.pre-commit-config.yaml

```diff
@@ -87,3 +87,10 @@ repos:
   hooks:
   - id: check-manifest
     stages: [manual]
+
+- repo: https://github.com/asottile/pyupgrade
+  rev: v2.11.0
+  hooks:
+  - id: pyupgrade
+    args: [--py36-plus]
+    exclude: tests/data/packages/BrokenEmitsUTF8/setup.py
```

Comment:
I'm fine with the actual changes below, but -1 on applying this tool automatically.

Reply:
Yea, let's do this -- and we'll tackle the broader question of automated code formatting there.